shared/js/dataDriver.js

```diff
@@ -413,7 +423,14 @@ loop.DataDriver = function() {
           break;
 
         case "page":
-          dispatchAction = "AddedPage";
+          if (typeof data.value !== "undefined") {
+            dispatchAction = "AddedPage";
```

Comment:
Probably. Otherwise the UI wouldn't have a good reference to which tile to delete. ;) Similarly, `DeletedPage` should have a `pageId` so that if someone else deletes a page, other open table of contents can update appropriately.